run_pipeline.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+##################
+# NF1 Inactivation Classifier for Glioblastoma
+# Gregory Way 2016
+##################
+
+##################
+# Step 0 - Install R Packages and Download Data
+##################
+Rscript install.R
+
+./download_data.sh
+
+##################
+# Step 1 - Process Training Data
+##################
+python scripts/describe_data.py \
+        --tissue 'GBM' \
+        --gene 'NF1' \
+        --type 'Silent'
+
+# Build X matrix
+python scripts/build_x.py \
+        --tissue 'GBM'
+
+# Process RNAseq data
+python scripts/process_rnaseq.py \
+        --tissue 'GBM' \
+        --scale 'zscore'
+
+# Build Y matrix
+python scripts/build_y.py \
+        --gene 'NF1' \
+        --tissue 'GBM' \
+        --y-origin 'data/Y/' \
+        --filter 'Silent' \
+        --unclass
+
+##################
+# Step 2 - Cross Validation
+##################
+# Parameter Sweep Values
+alpha_param='0.1,0.15,0.2,0.25,0.3,0.4,0.5,0.7,0.9'
+l1_ratio='0.1,0.15,0.18,0.2,0.22,0.24,0.26,0.28,0.3,0.4,0.5,0.6,0.7,0.8,0.9'
+
+# Performs cross validation for the parameter sweep and outputs figures
+python scripts/nf1_classifier.py \
+        --gene 'NF1' \
+        --tissue 'GBM' \
+        --output-file 'results/gbm_elasticnet_cv.tsv' \
```

Comment:
yeah, I think so. I don't know of a solution for doing this inside a bash script - but I should look into it...could be important